sdk/program/src/vote/instruction.rs

```diff
@@ -191,6 +191,10 @@ impl VoteInstruction {
         )
     }
 
+    pub fn is_tower_sync(&self) -> bool {
```

Comment:
@lidatong FYI we add a function here in case you need to replicate it on the FD side.

Reply:
if these instructions need to be 1:1 for FD's implementation, could we just directly use `match` where we need in our client? It's not really part of the program, just our filtering logic, right?

Reply:
fair point, then we don't have the maintenance burden of exposing it as part of the public API either.